client/my-sites/plans/current-plan/jetpack-product-install/index.tsx

```diff
@@ -233,20 +255,44 @@ export class JetpackProductInstall extends Component {
 	}
 }
 
+function mapStateToProps( state ) {
+	const siteId = getSelectedSiteId( state );
+	const queryArgs = getCurrentQueryArguments( state );
+
+	const installQuery: string[] =
+		// eslint-disable-next-line no-nested-ternary
+		typeof queryArgs === 'object' && 'install' in queryArgs
+			? Array.isArray( queryArgs.install )
+				? queryArgs.install
+				: [ queryArgs.install ]
+			: [];
+
+	const requestedInstalls: PluginSlug[] = installQuery.includes( 'all' )
```

Comment:
In Jetpack dash there are some special keywords used for installing plugins called "scan" and "backups":

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/87168/60321654-1a742f00-9986-11e9-8353-7eec39685286.png">

(those "set up" buttons show up in dash when Akismet/Vaultpress are disabled when we already have a plan)

Suppose we don't need to add those terms here and we can handle that in route-redirections?

Reply:
Yep, I'll handle that in an independent PR